Extend/Controller/Admin/PaymentMain.php

```diff
@@ -43,6 +44,14 @@ class PaymentMain extends PaymentMain_parent
      */
     private $_bIsSavePossible;
 
+    /**
+     * Tells if information about currency settings should be shown.
+     * The information is shown any time merchant adds a currency which has empty configuration fields.
+     *
+     * @since 1.2.0
+     */
+    private $_bCurSettingInfo;
```

Comment:
I would prefer a more descriptive name as I would not have understood it without reading the comment. Something like `bShowCurrencyConfigInfoMsg` or similar. I would not relate "Cur" to currency.

Reply:
Variable name can't be longer than 20 characters. Maybe we can try with "$_bCurrencyHelpMsg". Sounds good to you?

Reply:
Ok, then I would propose `_bShowCurrencyHelp` to indicate the action that will be triggered depending on the bool.

Reply:
👍